rules/windows/process_access/sysmon_load_undocumented_autoelevated_com_interface.yml

```diff
@@ -0,0 +1,29 @@
+title: Load Undocumented Autoelevated COM Interface
+id: fb3722e4-1a06-46b6-b772-253e2e7db933
+status: experimental
+description: COM interface (EditionUpgradeManager) that is not used by standard executables.
+references:
+    - https://www.snip2code.com/Snippet/4397378/UAC-bypass-using-EditionUpgradeManager-C/ 
+    - https://gist.github.com/hfiref0x/de9c83966623236f5ebf8d9ae2407611
+tags:
+    - attack.defense_evasion
+    - attack.privilege_escalation
+    - attack.t1548.002
+author: oscd.community, Dmitry Uchakin
+date: 2020/10/07
+logsource:                  
+    category: process_access
+    product: windows        
+detection:
+    selection:
+        CallTrace|contains: '*editionupgrademanagerobj.dll*'
```

Comment:
could you please remove `*` from the `'*editionupgrademanagerobj.dll*'` as it is already supposed by the `contains`?

Reply:
thank you!